near-sdk/src/utils/mod.rs

```diff
@@ -86,8 +91,13 @@ impl PendingContractTx {
         )
     }
 
-    pub fn new_from_bytes(receiver_id: &str, method: &str, args: Vec<u8>, is_view: bool) -> Self {
-        Self { receiver_id: receiver_id.to_string(), method: method.to_string(), args, is_view }
+    pub fn new_from_bytes(
+        receiver_id: AccountId,
+        method: &str,
+        args: Vec<u8>,
+        is_view: bool,
+    ) -> Self {
+        Self { receiver_id: receiver_id, method: method.to_string(), args, is_view }
```

Comment:
nit: :bulb: use field shorthand syntax

Reply:
ah, yeah definitely going to fix clippy and add to CI before next release